core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -232,6 +225,13 @@ private[spark] class Executor(
     }
 
     override def run(): Unit = {
+      val taskId = taskDesc.taskId
+      val attemptNumber = taskDesc.attemptNumber
+      val taskName = taskDesc.name
+      val taskFiles = taskDesc.taskFiles
+      val taskJars = taskDesc.taskJars
+      val taskProps = taskDesc.taskProperties
```

Comment:
If use `taskDesc.taskId`, we need to modify a lot of code. eg:
```scala
val errMsg =
              s"${releasedLocks.size} block locks were not released by TID = $taskId:\n" +
                releasedLocks.mkString("[", ", ", "]")
```
The length of this line of  `s"${releasedLocks.size} block locks were not released by TID = ${taskDesc.taskId}:\n"` is greater than 100

How do I move the code to:

```scala
  class TaskRunner(
      execBackend: ExecutorBackend,
      val taskDesc: TaskDescription)
    extends Runnable {

    val taskId = taskDesc.taskId
    val attemptNumber = taskDesc.attemptNumber
    val taskName = taskDesc.name
    val taskFiles = taskDesc.taskFiles
    val taskJars = taskDesc.taskJars
    val taskProps = taskDesc.taskProperties
```